completions/bash/multipass

```diff
@@ -90,8 +160,30 @@ _multipass_complete()
                 _filedir
                 return
             ;;
-            # TODO: add "--network" here
+            "--network")
+                compopt -o nospace
+                # TODO: Do use spaces after the completion of an interface name.
+                _multipass_networks
+                opts="${opts} id= mode= mac="
         esac
+
+    # We need to complete the option pairs in two steps. One, when the last input char was an equals sign. Two,
+    # when the user starts to input something, because prev changes when she starts to type a value.
+    elif [[ "${cur}" == '=' ]] && [[ "${prev2}" == ',' || "${prev2}" == '--network' ]] ; then
```

Comment:
Right, this will work until we add another option using var=val pairs. For instance, if we add completions to `set` and `get`.

Since I implemented searching backwards `--network`, this implementation will be a small refactor. And I think it's worth the effort because we'll reuse backwards searching when needed.

Reply:
Done, I tried to refactor as much as reasonable, in order to add some useful functions. And left a note where a function can be factored when needed.